lib/main.js

```diff
@@ -394,7 +394,7 @@ function ConfigureMathJax() {
               jax.CHTML.ex = ex = (data||defaults).ex;
               jax.CHTML.em = jax.CHTML.outerEm = em = ex / CHTML.TEX.x_height; // scale ex to x_height
               jax.CHTML.cwidth = width / em;
-              jax.CHTML.lineWidth = (linebreak ? width / em : CHTML.BIGDIMEN);
+              jax.CHTML.lineWidth = (linebreak ? width / em : 1000000);
```

Comment:
Why is that replaced. It does not differ from the value in chtml.
Either way, a magic number should be assigned to a named constant.